### ISSUE TITLE
As Buyer when I arrive at the homepage of the site/application, I need the ability to search for equipment by filtering on a set of criteria (i.e. Aerial Lift Type, Sub-Type, Reach, Proximity, Price).

### ISSUE THREAD

**Issue by Dan-Madden** [Alex-Tideman/equiplogue-user-stories#2]

**Comment by Dan-Madden** [comment]
The 'Data Navigation' tab has a subset of data (still work in progress) to demonstrate the filtering of certain search criteria. For example on the  cars.com home page search, when you select 'Ford' in the 'Make' dropdown, the models narrow down to only Ford models.

**Comment by Alex-Tideman** [comment]
Makes sense, but what should the Data Navigation tab look like? Where should it live on the page (top, side)?

**Comment by Alex-Tideman** [comment]
Do we also want a map of the units? We would need to know the location of the unit (either city or lat long).
